win32ss/gdi/ntgdi/freetype.c

```diff
@@ -2994,39 +2988,102 @@ static unsigned int get_bezier_glyph_outline(FT_Outline *outline, unsigned int b
     return needed;
 }
 
-static INT
-IntRequestFontSize(PDC dc, PFONTGDI FontGDI, LONG Width, LONG Height)
+static FT_Error
+IntRequestFontSize(PDC dc, PFONTGDI FontGDI, LONG lfWidth, LONG lfHeight)
```

Comment:
Rename is to better match wine?

Reply:
I think not. calc_ppem_for_height is different from our IntRequestFontSize in design.